proxy/grpcproxy/health.go

```diff
@@ -43,3 +51,25 @@ func checkHealth(c *clientv3.Client) etcdhttp.Health {
 	}
 	return h
 }
+
+func checkProxyHealth(c *clientv3.Client) etcdhttp.Health {
+	h := etcdhttp.Health{Health: "true"}
+	ctx, cancel := context.WithTimeout(c.Ctx(), time.Second)
```

Comment:
Can we embed `checkHealth` and add extra watch checks?

Reply:
updated.thanks.